src/Nox.Types.EntityFramework/Abstractions/NoxDatabaseConfigurator.cs

```diff
@@ -39,20 +39,26 @@ public abstract class NoxDatabaseConfigurator : INoxDatabaseConfigurator
 
         public virtual void ConfigureEntity(
             NoxSolutionCodeGeneratorState codeGeneratorState,
-            EntityTypeBuilder builder,
+            IEntityBuilderAdapter builder,
             Entity entity,
-            IReadOnlyList<EntityRelationshipWithType> relationshipsToCreate)
+            NoxSolution noxSolution,
```

Comment:
the codeGeneratorState already contains a Solution no need to input it again

Reply:
Done